norse/torch/functional/test/test_iaf.py

```diff
@@ -0,0 +1,27 @@
+import torch
+from norse.torch.functional.iaf import (
+    IAFFeedForwardState,
+    IAFParameters,
+    IAFState,
+    iaf_step,
+    iaf_feed_forward_step,
+)
+
+
+def test_iaf_feed_forward_step_batch():
+    x = torch.ones(2, 1)
+    s = IAFFeedForwardState(v=torch.zeros(2, 1))
+
+    z, s = iaf_feed_forward_step(x, s)
+    assert z.shape == (2, 1)
```

Comment:
I would recommend adding a one line test to validate that the spikes/state actually change over time. Mostly for regression purposes